src/components/Header/SearchingBar/SearchingBar.component.jsx

```diff
@@ -1,10 +1,66 @@
 import React from 'react';
-import { Search, SearchContainer } from './SearchingBar.styles';
+import { useHistory, useParams } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import { Search, SearchContainer, Label } from './SearchingBar.styles';
+import { useVideo } from '../../../providers/Video';
 
 function SearchingBar() {
+  const { search, setSearch, setVideos, setAlert } = useVideo();
+  const history = useHistory();
+  const { videoId } = useParams();
+
+  const searchVideos = async (e) => {
```

Comment:
This code ca be improved, I think the function is doing quite a bit, maybe create a custom hook to search can be a great idea

Reply:
agree